_posts/zh_CN/pages/2016-01-01-about.md

```diff
@@ -5,13 +5,31 @@ permalink: /zh_CN/about/
 type: pages
 layout: page
 lang: zh_CN
-version: 1
+version: 2
 ---
 
+## 关于我们
+
 比特币核心是一个[开源项目]，主要负责维护和发布名为“比特币核心”的比特币客户端软件。
 
 中本聪在发布著名的[白皮书](/bitcoin.pdf)之后又发布了一个比特币软件客户端，而比特币核心正是这个客户端的后继者。
 
 比特币核心包含有全节点验证和比特币钱包。同时该项目还在维护一些相关软件如密码库[libsecp256k1](https://github.com/bitcoin/secp256k1)和其他[GitHub](https://github.com/bitcoin)义务翻译.
 
-所有人都可以[资助比特币核心团队](/zh_CN/contribute/).
\ No newline at end of file
+所有人都可以[资助比特币核心团队](/zh_CN/contribute/).
+
+### 团队
+
+核心比特币项目有一个包括许多短期贡献者的大型开源开发者社区。还有更多的人在调查研究、同行评价、测试、文档和翻译方面做出贡献.
+此页面列出了本项目一些更像全职开发的人员，以及拥有一些管理角色的成员.
+
+### 维护人员
+
+项目维护人员有提交权限，并负责合并贡献者开发的补丁。在合并团队认可的补丁的过程中，他们扮演了看门人的角色. 它们还充当最终检查员，以确保补丁是安全的，并且与项目目标一致. 维护人员的角色是由项目贡献者协商决定的.
+  
+**贡献者**
```

Comment:
I know you're trying to make this move only, but this means 'Contributors' and should be changed to h3 to match the english version.